views/layouts/styleguide.njk

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="{{ locale }}" class="no-js locale--{{ locale }}{% if highContrast %} contrast--high{% endif %}">
```

Comment:
Would it be overkill to make this a shared template too? Feels like these are quite "core" styles so might be a bit confusing if they're in two places (even for an exceptional page like this).

Reply:
(to be specific, I mean the `<html>` element)